smart_card_connector_app/src/window-main.js

```diff
@@ -58,7 +58,34 @@ GSC.ConnectorApp.Window.AppsDisplaying.initialize();
 GSC.ConnectorApp.Window.DevicesDisplaying.initialize();
 GSC.ConnectorApp.Window.LogsExporting.initialize();
 
-GSC.PopupWindow.Client.setExplicitWindowTitle();
 GSC.PopupWindow.Client.showWindow();
 
+/** @const */
+var DATA_I18N_STR = 'data-i18n';
+
+/** @const */
+var DATA_I18N_ARIA_LABEL_STR = 'data-i18n-aria-label';
+
+function setElementI18nText(element) {
+  var i18nId = element.getAttribute(DATA_I18N_STR);
+  var translatedText = chrome.i18n.getMessage(i18nId);
```

Comment:
Please add assertions that the returned value is not undefined.
It's a bad error if somebody mistypes the message name; and the consequence is that the "undefined" string would be rendered, which would confuse the user.